arangod/Cluster/HeartbeatThread.cpp

```diff
@@ -545,7 +596,7 @@ void HeartbeatThread::runSingleServer() {
           // preemtily remove the transient entry from the agency
           _agency.setTransient(transientPath, VPackSlice::emptyObjectSlice(), 0);
```

Comment:
this seems to be incomplete. In devel `_agency.setTransient(transientPath, VPackSlice::emptyObjectSlice(), 0);` is not there anymore